specs/models/MasterCollections.v1.json

```diff
@@ -0,0 +1,52 @@
+{
+  "title": "MasteringCollections.v1",
```

Comment:
I think the file and title should have the same name - "MasteringCollections" seems appropriate.

Reply:
The UX designs are using "Match" and "Merge" in the UI, so I recommended leaving off the "ing" for related files in the code. Should we do the same for Mastering, so "Master"? I see "Load" and "Map" and "Curate" in the UI as well. Shorter is better.

Reply:
Existing DH core uses mapping/matching/merging/mastering.